neuraxle/base.py

```diff
@@ -379,25 +441,17 @@ def inverse_transform(self, processed_outputs):
         """
         return processed_outputs
 
-    def inverse_transform_one(self, processed_output):
-        """
-        Do nothing - return the same data.
-
-        :param processed_output: the data to process
-        :return: the ``data_output``, unchanged.
-        """
-        return processed_output
-
 
 class TruncableSteps(BaseStep, ABC):
 
     def __init__(
             self,
             steps_as_tuple: NamedTupleList,
             hyperparams: HyperparameterSamples = dict(),
-            hyperparams_space: HyperparameterSpace = dict()
+            hyperparams_space: HyperparameterSpace = dict(),
+            hasher: Hasher = None
```

Comment:
maybe force hasher prop here without defaut value (see comments above)

Reply:
I'd use the ID hasher as the pipeline will use that as a starting point for the data. 

Potential bug I just thought of: what if we have nested pipelines, will the inner pipeline have access to the rehashed IDs of the new data ? It'd need to override the handle methods to obtain it.